### ISSUE TITLE
Profile form not showing date control

### ISSUE THREAD

**Issue by jamesrwelch** [MauroDataMapper/mdm-ui#166]
A field with 'date' data type in the profile is not resulting in the date-picker control.

**Comment by jamesrwelch** [comment]
Ok, the dataType is coming through and being lower-cased just fine - reassigning to somebody competent

**Comment by gammonpeter** [comment]
Just tested with latest code and appears to be working will re test once issue with backend is resolved

**Comment by gammonpeter** [comment]
Just retested using profile provided and working as expected

**Comment by gammonpeter** [comment]
Fails on Safari browsers

**Comment by pjmonks** [comment]
May have to consider working in tandem with #224

**Comment by pjmonks** [comment]
I think the current status of this was that we need to upgrade to the latest version of the Angular framework first. The current npm dependencies in place are not correct, and updating some of them manually will cause issues elsewhere in the app.

That has been marked as "on hold" so that #224 has been resolved first.

**Comment by gammonpeter** [comment]
Should now be unblocked with angular 12 update

**Comment by pjmonks** [comment]
@gammonpeter Please could I get a progress update on this issue? The last I heard about it (several months ago) was that a different date picker control was trying to be used to resolve this, but that's when the npm module dependencies got in the way. I'm not sure what current state this issue is in.

**Comment by gammonpeter** [comment]
No more progress then before, a date picker which is compatible with Safari needs to be chosen and implemented
